source/globio.rst

```diff
@@ -1,5 +1,5 @@
-InVEST GLOBIO Model
-===================
+GLOBIO

```

Comment:
I wonder if InVEST GLOBIO is a good distinction from THE GLOBIO model? Just to reinforce this is a modified version of GLOBIO and not a repackage of that model.